index.html

```diff
@@ -2,14 +2,37 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <link rel="stylesheet" href="index.css" />
+  <link rel="stylesheet" href="index.css"/>
   <title>Portfolio</title>
 </head>
-
 <body>
+  <header class="header">
+    <div class="container">
+      <a class="logo" href="#">My Logo</a>
+      <nav class="navbar">
+        <div class="hamburger-menu">
+          <div class="line line-1"></div>
+          <div class="line line-2"></div>
+          <div class="line line-3"></div>
+        </div>
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#section1-nav" class="nav-link">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#section2-nav" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact-section_nav" class="nav-link">Contact</a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
### Your mobile NAV implementation is beautiful!!! However:

![image](https://user-images.githubusercontent.com/57421823/151876662-ee40877b-30c9-48cc-a8f6-8db2ab03bfe4.png)

The project requirements say otherwise 😢.  

The reason that we are requiring you to closely follow the designs in Figma, is that in this particular module, you are learning to implement a given design into a software solution.  Think of the Figma as a client's business requirements.  Later in the program, you will be given of course a lot more creative freedom 👍 

Therefore:

- [ ]  Please ensure the mobile menu follows the Figma design _very closely_:

![image](https://user-images.githubusercontent.com/57421823/151877373-e7cdfd4b-d765-4d1d-9ef9-be6e2d7c4fc9.png)



- [ ] Your nav links point to the wrong sections of the document.  Kindly fix them so they point to the right sections, that way, the user experience will be a lot better.

- [ ] OPTIONAL: If you want to make your header a sticky so it can be viewed at any point of the page, I suggest you keep its background because otherwise it gets lost:

- [ ] OPTIONAL: You can keep the animations! They are beautiful.

![image](https://user-images.githubusercontent.com/57421823/151878126-199016a5-770d-4f67-9143-58b0ecaa7a77.png)


Again, please remember to stick to the given template and that later in the program, you will be given a lot more creative freedom!! 💪